src/NetherPortalScanner.cpp

```diff
@@ -296,7 +296,7 @@ void cNetherPortalScanner::OnDisabled(void)
 	}
 
 	FLOGD("Placing player at {0}", Position);
-	m_Entity->MoveToWorld(m_World, Position, true);
+	m_Entity->MoveToWorld(m_World, Position, true, false);
```

Comment:
There are also the end portal cases to consider.

Reply:
As far as I can tell, these cases already disable respawn packets explicitly.

Reply:
Ah you're right, I was forgetting there are multiple signatures.

Reply:
I wish there weren't so many, but what's done is done.